<launch>
   <arg name="model" default="maelstrom"/>
   <arg name="model_states_topic" default="/gazebo/model_states"/>
   <arg name="six_dof_topic" default="/auv_gnc/trans_ekf/six_dof"/>
   <arg name="imu_topic" default="/state/imu"/>
   <arg name="dvl_topic" default="/state/dvl"/>
   <arg name="depth_topic" default="/state/depth"/>

   <node name="inertial_sensors" pkg="riptide_gazebo" type="inertial_sensors" output="screen">
      <param name="model" value="$(arg model)"/>
      <param name="model_states_topic" value="$(arg model_states_topic)"/>
      <param name="six_dof_topic" value="$(arg six_dof_topic)"/>
      <param name="imu_topic" value="$(arg imu_topic)"/>
      <param name="dvl_topic" value="$(arg dvl_topic)"/>
      <param name="depth_topic" value="$(arg depth_topic)"/>
   </node>

</launch>